README-pl.md

```diff
@@ -0,0 +1,83 @@
+# Dokumentacja Kubernetes
+
+[![Build Status](https://api.travis-ci.org/kubernetes/website.svg?branch=master)](https://travis-ci.org/kubernetes/website)
+[![GitHub release](https://img.shields.io/github/release/kubernetes/website.svg)](https://github.com/kubernetes/website/releases/latest)
+
+Witamy!
+
+W tym repozytorium znajdziesz wszystko, co potrzebujesz do zbudowania [strony internetowej Kubernetes wraz z dokumentacją](https://kubernetes.io/). Bardzo nam miło, że chcesz wziąć w tym udział!
+
+## Twój wkład w dokumentację
+
+Możesz kliknąć w przycisk **Fork** w prawym górnym rogu ekranu, aby stworzyć kopię tego repozytorium na swoim koncie GitHub. Taki rodzaj kopii (odgałęzienia) nazywa się *fork*. Zmieniaj w nim, co chcesz, a kiedy będziesz już gotowy/a przesłać te zmiany do nas, przejdź do swojej kopii i stwórz nowy *pull request*, abyśmy zostali o tym poinformowani.
+
+Po stworzeniu *pull request*, jeden z recenzentów Kubernetes podejmie się zadania przekazania jasnych wskazówek pozwalających podjąć konkretne działania. Na Tobie, jako właścicielu *pull request*, **spoczywa odpowiedzialność za wprowadzenie poprawek zgodnie z uwagami recenzenta Kubernetes.** Może też się zdarzyć, że swoje uwagi zgłosi więcej niż jeden recenzent, lub że recenzję będzie robił ktoś inny, niż ten, kto został przydzielony na początku. W niektórych przypadkach, jeśli zajdzie taka potrzeba, recenzent może poprosić dodatkowo o recenzję jednego z [recenzentów technicznych Kubernetes](https://github.com/kubernetes/website/wiki/Tech-reviewers). Recenzenci zrobią wszystko, aby odpowiedzieć sprawnie, ale konkretny czas odpowiedzi zależy do różnych czynników.
+
+Więcej informacji na temat współpracy przy tworzeniu dokumentacji znajdziesz na stronach:
+
+* [Start contributing](https://kubernetes.io/docs/contribute/start/)
+* [Staging Your Documentation Changes](http://kubernetes.io/docs/contribute/intermediate#view-your-changes-locally)
+* [Using Page Templates](http://kubernetes.io/docs/contribute/style/page-templates/)
+* [Documentation Style Guide](http://kubernetes.io/docs/contribute/style/style-guide/)
+* [Localizing Kubernetes Documentation](https://kubernetes.io/docs/contribute/localization/)
+
+## Różne wersje językowe `README.md`
+|  |  |
+|---|---|
+|[French README](README-fr.md)|[Korean README](README-ko.md)|
+|[German README](README-de.md)|[Portuguese README](README-pt.md)|
+|[Hindi README](README-hi.md)|[Spanish README](README-es.md)|
+|[Indonesian README](README-id.md)|[Chinese README](README-zh.md)|
+|[Japanese README](README-ja.md)|[Polish README](README-pl.md)|
+|||
+
+## Jak uruchomić serwis internetowy lokalnie przy pomocy Dockera
```

Comment:
`Jak wygenerować dokumentację lokalnie` feels a bit less awkward, don't you think?

Reply:
You are right, but it depends on how much we want to follow the original text.

Reply:
What about: "Jak uruchomić lokalnie stronę z dokumentacją przy pomocy Dockera?"

Reply:
Based on English translation, the text is "Running the website locally using Docker". Earlier there's "(...)the Kubernetes website and documentation", which was translated as "strony internetowej Kubernetes wraz z dokumentacją".

Why not just "Jak uruchomić stronę internetową Kubernetes lokalnie przy pomocy Dockera"?

Reply:
@nvtkaszpir was not happy about the original translation.